perfkitbenchmarker/benchmark_spec.py

```diff
@@ -155,6 +157,12 @@ def __init__(self, benchmark_module, benchmark_config, benchmark_uid):
         self.config.vm_groups if self.config.relational_db is None else
         relational_db.VmsToBoot(self.config.relational_db.vm_groups))
 
+    self.vpn_service = None
+    self.vpns = {}  # dict of vpn's
+    self.vpngws = {}  # dict of vpn gw's
```

Comment:
Suggest renaming to vpn_gws for readability - same for the lock as well.

Reply:
Updated for variable names: 's/vpngw/vpn_gw/g' basically, but left function and class names as is.